src/main/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource.java

```diff
@@ -383,12 +385,13 @@ public OffenderAssessments getAssessmentsByCrn(
         })
     @PostMapping(path = "/offenders/crn/{crn}/tier/{tier}")
     @PreAuthorize("hasRole('ROLE_MANAGEMENT_TIER_UPDATE')")
-    public OffenderDetail updateTier(
+    public ResponseEntity updateTier(
```

Comment:
Very minor, I wonder if this was `void` it would just work and return 200 with no body??